sites/fast-website/src/app/components/card-section/index.ts

```diff
@@ -0,0 +1,14 @@
+import { customElement } from "@microsoft/fast-element";
+import { CardSectionTemplate as template } from "./card-section.template";
+import { CardSectionStyles as styles } from "./card-section.styles";
+import { FASTCard as Card } from "@microsoft/fast-components";
+
+@customElement({
+    name: "card-section",
```

Comment:
could site be added to this name? such as site-card-section

Reply:
Done